.github/workflows/build_test_ci.yml

```diff
@@ -13,6 +13,7 @@ on:
     paths-ignore:
       - '**/**.md'
       - 'docs/**'
+  workflow_dispatch:
```

Comment:
Question: What is this for? Why does it have no inputs?

Reply:
This allows us to trigger a build from the actions tab as well, it doesn't need any specific inputs

Reply:
Oh super cool!